src/plugins/intel_gpu/src/graph/impls/onednn/primitive_onednn_base.h

```diff
@@ -482,6 +482,31 @@ struct typed_primitive_onednn_impl : public typed_primitive_impl<PType> {
         return args;
     }
 
+    virtual std::unordered_map<int, dnnl::memory> get_arguments(typed_primitive_inst<PType>& instance, kernel_arguments_data& mem_args) const {
```

Comment:
I think this is only for reorder. What about adding an assertion that checks `inputs.size() == 1` and `outputs.size() == 1`?

Reply:
Add assert lines for that.